build/CMakeHelpers/ParseCsProjForCsProjs.sh

```diff
@@ -6,7 +6,7 @@
 # We use this to help dynamically build up the set of dependencies for our
 # CMakeLists.txt wrappers around .csproj files.
 
-cat *.csproj 2>/dev/null \
+cat *.csproj *.proj 2>/dev/null \
```

Comment:
Let's us add `ProjectReference`s as dependencies for `.proj` files as well (see `Mlos.NetCore.Components.Packages/CMakeLists.txt`)